test/engagement.e2e-spec.ts

```diff
@@ -357,14 +358,18 @@ describe('Engagement e2e', () => {
     expect(updated.status).toBe(EngagementStatus.Rejected);
   });
 
-  it('updates internship engagement', async () => {
+  it.only('updates internship engagement', async () => {
```

Comment:
```suggestion
  it('updates internship engagement', async () => {
```

Reply:
ok making a mental note for myself to always double check this 😬

Reply:
We need a lint rule lol

Reply:
There's a few in the market. https://www.npmjs.com/package/eslint-plugin-no-only-tests?